configure

```diff
@@ -104,7 +104,7 @@ else
   sed -e "s|@openmp_cflags@|\$(SHLIB_OPENMP_CFLAGS)|" src/Makevars.in > src/Makevars
 fi
 # retain user supplied PKG_ env variables, #4664. See comments in Makevars.in too.
-sed -i "s|@PKG_CFLAGS@|$PKG_CFLAGS|" src/Makevars
-sed -i "s|@PKG_LIBS@|$PKG_LIBS|" src/Makevars
+sed -i "" "s|@PKG_CFLAGS@|$PKG_CFLAGS|" src/Makevars
```

Comment:
I thought I remembered `sed -i` requires a non-default `sed` on Mac.

To be extensible I think we need to `sed` to a temp file, then `mv` it afterwards & purge the temp

Reply:
Yes, I agree. That may be the reason of the CI fails...